src/config/constants.ts

```diff
@@ -68,6 +66,12 @@ export const TOP_NUMBER_OF_ITEMS_TO_DISPLAY = 10;
 
 export const DEFAULT_REQUEST_SAMPLE_SIZE = 5000;
 
+export enum Context {
+  Builder = 'builder',
+  Player = 'player',
+  Library = 'library',
+  Analytics = 'analytics',
+}
```

Comment:
If I am correct you do not need the "analytics" key in actions ? does it make sens to see actions generated in analytics in analytics ? 
```suggestion
export const ActionViewContext = {
    Builder: Context.Builder,
    Player: Context.Player,
    Library: Context.Library
}
```